todoman/ui.py

```diff
@@ -251,8 +251,23 @@ def detailed(self, todo):
         return rv
 
     def format_date(self, date):
+        """
+        Returns date in the following format:
+        * if date == today or tomorrow: "Today" or "Tomorrow"
+        * else: return a string representing that date
+        """
         if date:
-            rv = date.strftime(self.date_format)
+            date_tomorrow = datetime.today().date() + timedelta(days = 1)
```

Comment:
Instead of calling `datetime.today()` here, please use `self.now`. This is because we might want to replace `self.now` with a fixed value in the testsuite.